buildenv/jenkins/variables/defaults.yml

```diff
@@ -537,7 +537,7 @@ x86-64_windows:
     os: 'windows'
     url:
```

Comment:
I think this whole url section can be removed so the default in the boot_jdk_default is used.